src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/AddOrRemoveRptTransactionsController.java

```diff
@@ -0,0 +1,127 @@
+package uk.gov.companieshouse.web.accounts.controller.smallfull;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.view.UrlBasedViewResolver;
+import org.springframework.web.util.UriTemplate;
+import uk.gov.companieshouse.api.ApiClient;
+import uk.gov.companieshouse.api.model.accounts.smallfull.SmallFullApi;
+import uk.gov.companieshouse.web.accounts.annotation.NextController;
+import uk.gov.companieshouse.web.accounts.annotation.PreviousController;
+import uk.gov.companieshouse.web.accounts.api.ApiClientService;
+import uk.gov.companieshouse.web.accounts.controller.BaseController;
+import uk.gov.companieshouse.web.accounts.exception.ServiceException;
+import uk.gov.companieshouse.web.accounts.model.relatedpartytransactions.AddOrRemoveRptTransactions;
+import uk.gov.companieshouse.web.accounts.service.smallfull.RptTransactionService;
+import uk.gov.companieshouse.web.accounts.service.smallfull.SmallFullService;
+
+import javax.servlet.http.HttpServletRequest;
+
+@Controller
+@NextController(OffBalanceSheetArrangementsQuestionController.class)
+@PreviousController(RelatedPartyTransactionsQuestionController.class)
+@RequestMapping("/company/{companyNumber}/transaction/{transactionId}/company-accounts/{companyAccountsId}/small-full/note/add-or-remove-transactions")
+public class AddOrRemoveRptTransactionsController extends BaseController {
```

Comment:
Sorry, just noticed we should be implementing conditional controller here and overwriting the WillRender method